### PR TITLE
Update config

### PR DESCRIPTION
**Pull request by chiwa0707**

### CODE REVIEW

File: .tx/config [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [main]
-host = https://www.transifex.com
+host = https://www.translation-011x.com
```
Comment (Diapolo) [inline .tx/config]: What the hell is that?

**Comment by sipa** [comment]
What is this?

**Comment by chiwa0707** [comment]
Can you explain it to me how to do this ?

On Sunday, June 14, 2015, Pieter Wuille <notifications@github.com> wrote:

> What is this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/6279#issuecomment-111825416>.
>


-- 
*- Vickiee :D*

**Comment by sipa** [comment]
I have no clue what the "this" is that you are trying to do.

**Comment by chiwa0707** [comment]
I think I signed up for the wrong thing, a friend of mine buys bitcoins and
I want to buy one instead of make one.

On Sunday, June 14, 2015, Pieter Wuille <notifications@github.com> wrote:

> I have no clue what the "this" is that you are trying to do.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/6279#issuecomment-111826220>.
>


-- 
*- Vickiee :D*

**Comment by chiwa0707** [comment]
Huh ?

On Sunday, June 14, 2015, Pieter Wuille <notifications@github.com> wrote:

> Closed #6279 <https://github.com/bitcoin/bitcoin/pull/6279>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/6279#event-330429592>.
>


-- 
*- Vickiee :D*

**Comment by sipa** [comment]
What you did was send a request to change some of the code in the Bitcoin Core repository, the software package that runs most of Bitcoin's network infrastructure.

If you just want to buy bitcoins, you need to sign up for some exchange and/or find someone who wants to sell them to you. Search a bit on the internet. This is not the right place.

**Comment by chiwa0707** [comment]
Okay thank you

On Sunday, June 14, 2015, Pieter Wuille <notifications@github.com> wrote:

> What you did was send a request to change some of the code in the Bitcoin
> Core repository, the software package that runs most of Bitcoin's network
> infrastructure.
>
> If you just want to buy bitcoins, you need to sign up for some exchange
> and/or find someone who wants to sell them to you. Search a bit on the
> internet. This is not the right place.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/6279#issuecomment-111827557>.
>


-- 
*- Vickiee :D*

**Pull request closed without merging**